README.md

```diff
@@ -0,0 +1,174 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+*Web browser like  <a href='https://www.google.com/chrome/'>Google Chrome</a>
+
+*<a href='https://git-scm.com/'>Git</a>
+
+*<a href='https://github.com/'>GitHub</a>
+
+*<a href='https://code.visualstudio.com/'>Source Code Editor</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/JhonatanErosa/Project-1_Hello-Microverse
+
+
+### Install
+
+Install this dependencies:
+
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+*Clone the repo to your pc
+
+*Use the extension <a href='https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer'>Live Server
+</a> or open de .html file 
+
+### Run tests
+
+To run tests, run the following command:
+
+*HTML linters:
+  
+npx hint .
+
+*CSS linters:
+
+npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+<a href='https://github.com/'>GitHub</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Jhonatan Ali Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project support us with a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank the entire Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
```

Comment:
You added the FAQ header but you did not provide the answers to any FAQs. Since the FAQ section of the Readme file is totally optional, I think that you can remove it so that your project looks professional. 👍